lib/ulid/generate.rb

```diff
@@ -9,15 +9,18 @@ module Generate
 
     # returns the binary ULID as Base32 encoded string.
     def encode32
-      (hi, lo) = bytes.unpack("Q>Q>")
-      value = (hi << 64) | lo
+      high = bytes.unpack1("Q>")
```

Comment:
won't this cause a double scan of the bytes? What's the advantage of splitting this out?

Reply:
I believe the `unpack1` at least avoids array allocations but not sure how that compares to the double scan.

Looks like we [opted to keep it](https://github.com/abachman/ulid-ruby/pull/10#discussion_r1632590377) (presumably?) so I opted to retain it also for now.